azure_function/resource-graph-collector/publisher.properties

```diff
@@ -0,0 +1,3 @@
+[publisher]
+loki=True
+azure=False
```

Comment:
it's not clear to me how this is going to work. You plan to pass this file during the deployment in terraform?
I think we can make it much simpler, by adding 2 env vars:
`ENABLE_LOKI_PUBLISHER=true|false` and `ENABLE_AZURE_BLOB_PUBLISHER=true|false` 
and we drive the switches in the function app configuration. 

What you think?